src/index.d.ts

```diff
@@ -761,10 +761,21 @@ declare namespace admin.projectManagement {
     resourceName?: string;
   }
 
-  interface AndroidAppMetadata {
-    resourceName: string;
+  interface AppMetadata {
     appId: string;
-    displayName: string | null;
+    displayName?: string;
+    platform: AppPlatform;
+  }
+
+  enum AppPlatform {
+    PLATFORM_UNKNOWN = 'PLATFORM_UNKNOWN',
+    IOS = 'IOS',
+    ANDROID = 'ANDROID',
+  }
+
+  interface AndroidAppMetadata extends AppMetadata {
+    platform: AppPlatform.ANDROID;
+    resourceName: string;
```

Comment:
`resourceName` is a common field, so we should be able to include that in `AppMetadata`

Reply:
Also add `projectId` to `AppMetadata`